From 93bd1a481e13d7eb073b29fb51d0f98f41022c46 Mon Sep 17 00:00:00 2001
From: huangyifeng <huangyifeng@eswincomputing.com>
Date: Mon, 9 Sep 2024 10:29:47 +0800
Subject: [PATCH 167/223] fix:Synchronize configuration from eic7700_defconfig

Changelogs:
     Synchronize feature configuration from eic7700_defconfig to
eic7700_dbg_defconfig

Signed-off-by: huangyifeng <huangyifeng@eswincomputing.com>
---
 arch/riscv/configs/eic7700_dbg_defconfig | 7 +++----
 1 file changed, 3 insertions(+), 4 deletions(-)

diff --git a/arch/riscv/configs/eic7700_dbg_defconfig b/arch/riscv/configs/eic7700_dbg_defconfig
index 328c51187781..ed5d3e6ea9e3 100644
--- a/arch/riscv/configs/eic7700_dbg_defconfig
+++ b/arch/riscv/configs/eic7700_dbg_defconfig
@@ -32,12 +32,12 @@ CONFIG_PERF_EVENTS=y
 CONFIG_SOC_SIFIVE=y
 CONFIG_SOC_VIRT=y
 CONFIG_SMP=y
+CONFIG_HOTPLUG_CPU=y
 CONFIG_RISCV_SBI_V01=y
 # CONFIG_RISCV_BOOT_SPINWAIT is not set
-CONFIG_CMDLINE="earlycon=sbi console=ttyS0,115200n8 clk_ignore_unused cma_pernuma=0x2000000 disable_bypass=false"
+CONFIG_CMDLINE="earlycon=sbi console=ttyS0,115200n8 clk_ignore_unused cma_pernuma=0x2000000 disable_bypass=false firmware_class.path=/lib/firmware/eic7x/"
 CONFIG_CMDLINE_EXTEND=y
-CONFIG_PM_DEBUG=y
-CONFIG_PM_ADVANCED_DEBUG=y
+# CONFIG_SUSPEND is not set
 CONFIG_CPU_FREQ=y
 CONFIG_CPU_FREQ_STAT=y
 CONFIG_CPU_FREQ_DEFAULT_GOV_USERSPACE=y
@@ -799,7 +799,6 @@ CONFIG_CRYPTO_SHA512=m
 CONFIG_CRYPTO_DEV_VIRTIO=y
 CONFIG_CRC_ITU_T=y
 CONFIG_CRC7=y
-CONFIG_XZ_DEC=y
 CONFIG_DMA_CMA=y
 CONFIG_PRINTK_TIME=y
 CONFIG_CONSOLE_LOGLEVEL_DEFAULT=15
-- 
2.47.0

